x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_policy_create_extension.tsx

```diff
@@ -5,45 +5,535 @@
+      setInteractive(radioComprehensiveSelected);
+    }, [radioInteractiveSelected, radioComprehensiveSelected, setInteractive]);
+
+    const onChangeRadioEndpoint = useCallback(
+      (e) => {
+        setRadioEndpointOption(e.target.value);
+        setPreset(e.target.value);
+      },
+      [setPreset]
+    );
+    ///
+
+    const onChangeMalwareCheckbox = useCallback(() => {
+      setCheckboxMalwareChecked(!checkboxMalwareChecked);
+      setPreventions(checkboxMaliciousChecked, !checkboxMalwareChecked);
+    }, [checkboxMalwareChecked, checkboxMaliciousChecked, setPreventions]);
+
+    const onChangeRansomwareCheckbox = useCallback(() => {
+      setCheckboxMaliciousChecked(!checkboxMaliciousChecked);
+      setPreventions(!checkboxMaliciousChecked, checkboxMalwareChecked);
+    }, [checkboxMalwareChecked, checkboxMaliciousChecked, setPreventions]);
+
     return (
-      <>
-        <EuiSpacer size="m" />
-        <EuiCallOut data-test-subj="endpointPackagePolicy_create" iconType="iInCircle">
-          <EuiText size="s">
-            <p>
-              <FormattedMessage
-                id="xpack.securitySolution.endpoint.ingestManager.createPackagePolicy.endpointConfiguration"
-                defaultMessage="We'll save your integration with our recommended defaults. You can change this later by editing the Elastic Defend integration within your agent policy."
+      <EuiForm component="form">
+        <SelectConfigurationSettingsText>
+          <FormattedMessage
+            id="xpack.securitySolution.createPackagePolicy.stepConfigure.enablePrevention"
+            defaultMessage="Select configuration settings"
+          />
+        </SelectConfigurationSettingsText>
+        <QuickSettingInfo>
+          <FormattedMessage
+            id="xpack.securitySolution.createPackagePolicy.stepConfigure.quickSettingsTranslation"
+            defaultMessage="Use quick settings to configure the integration to {environments}. You can make changes to the configurations after you add it."
+            values={{
+              environments: (
+                <b>
+                  <FormattedMessage
+                    id="xpack.securitySolution.endpoint.ingestManager.createPackagePolicy.environments"
+                    defaultMessage="protect your tranditional endpoints or dynamic clound environments"
+                  />
+                </b>
+              ),
+            }}
+          />
+        </QuickSettingInfo>
+        <PaddingBottomText>
+          <FormattedMessage
+            id="xpack.securitySolution.createPackagePolicy.stepConfigure.selectEnvironmentTextTranslation"
+            defaultMessage="Select for what environment you would like to add the integration"
+          />
+        </PaddingBottomText>
+        <EuiSelect
+          id="selectIntegrationTypeId"
+          options={dropDownOptions}
+          value={dropdownValue}
+          onChange={(e) => onChangeDropdown(e)}
```

Comment:
Remove arrow function and just use the memoized callback - `onChangeDropdown`

Same comment for several others below.